### DISCUSSION TITLE
【两栏布局】使用两栏排版时，网页的排版与Notion要一致

### DISCUSSION THREAD

**Discussion by Angel0726** [tangly1024/NotionNext#2040] [Ideas]
**描述bug**
简单说明bug的现象、相关的错误提示、日志等
https://next.zeabur.app/article/a291e09c-3b09-484e-b9aa-2d895e9e3a89
![image](https://user-images.githubusercontent.com/32816780/227853323-d1652c3e-5f31-4b2d-bc9c-384a5e2a0f96.png)
https://www.notion.so/jeweled-trilby-556/Transformer-a291e09c3b09484eb9aa2d895e9e3a89
![image](https://user-images.githubusercontent.com/32816780/227853566-e34c137d-5fa5-4900-a7f4-2c6ce542b5da.png)

**复现步骤**
出现这个bug的操作步骤

**期望的正常结果**
希望按这个步骤，正常操作结果是什么
希望能够同意next主题与notion的排版，这样写博客的时候就会得到两个主题的一致性。目前next主题基本占满了屏幕，是不是可以把博客内容的左右留白弄小一点
![image](https://user-images.githubusercontent.com/32816780/227854128-fd71fae7-bab8-434b-aa20-b6ccc4e0a235.png)

**截图**
相关的页面，应该用结果

**环境**

- 操作系统: [例如.windows]
- 浏览器 [例如.edge]

**补充说明**
与问题相关的其它说明

**Comment by tangly1024** [comment]
OK， 正在处理

**Comment by tangly1024** [comment]
已提交补丁做了些许优化，但要完全一致排版比较困难：受主题的样式结构影响、不同屏幕尺寸的解决方案也不尽不同，只能是尽量模仿

**Comment by Angel0726** [comment]
> 已提交补丁做了些许优化，但要完全一致排版比较困难：受主题的样式结构影响、不同屏幕尺寸的解决方案也不尽不同，只能是尽量模仿

我更新看一下。行数跟notion的对应上了。但排版还有一些问题。不知道是不是行间距的问题；而且对比notion，可能每一行的字数比notion还要多一些，多的字数，可能之后也造成排版问题。
![image](https://user-images.githubusercontent.com/32816780/227868584-4abc58f6-936b-4ebf-b601-f1a61ea8e085.png)

所以有两种方案。
第一种：减少行间距，较少每行的字数，这样与notion完全保持一致
第二种：保持行间距，增加每一行的字数，减少行数。我对比了hexo主题，具体需要减少左右留白，增加每一行的字数，具体增加3个字（对于两栏博客）
